### PR TITLE
include includes hash in message show mobile endpoint

### PR DESCRIPTION
**Pull request by aherzberg**
## Summary
include includes hash in message show mobile endpoint

## Related issue(s)
https://github.com/department-of-veterans-affairs/va-mobile-app/issues/9327

## Testing done

- [ ] *New code is covered by unit tests*
- *Describe what the old behavior was prior to the change*
- *Describe the steps required to verify your changes are working as expected. Exclusively stating 'Specs run' is NOT acceptable as appropriate testing*
- *If this work is behind a flipper:*
  - *Tests need to be written for both the flipper on and flipper off scenarios. [Docs](https://depo-platform-documentation.scrollhelp.site/developer-docs/feature-toggles-guide#Featuretogglesguide-Backendexample).*
  - *What is the testing plan for rolling out the feature?*

## Screenshots
_Note: Optional_

## What areas of the site does it impact?
*(Describe what parts of the site are impacted and*if*code touched other areas)*

## Acceptance criteria

- [ ]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ]  No error nor warning in the console.
- [ ]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [ ]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature

## Requested Feedback

(OPTIONAL)_What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: modules/mobile/spec/request/messages_request_spec.rb
Comment (aherzberg) [inline modules/mobile/spec/request/messages_request_spec.rb:67]: message schema is shared with other modules and does not include the `includes` field so adding it would require other teams approval. Given the urgency, I made it pass by making it less strict

**Review by jperk51 — APPROVED** [review]
LGTM, but make sure to test in staging asap after merging.

**Review by RachalCassity — APPROVED** [review]

**Pull request merged**
